index.html

```diff
@@ -6,11 +6,21 @@
   </head>
   <body>
     <h1>Muay Thai</h1>
-    
+
+    <!-- 
+    Work in Progress
+    <nav>
+      <ul>
+  	    <li><a href="details.html">Styles<a></li>
+  	    <li><a href="images.html">Images<a></li>
+  	  </ul>
+    </nav>
+    -->
+
     <!-- Login Form -->
-	<form action="https://httpbin.org/post" method="post">
+	<form action="contact.html" method="post">

```

Comment:
This is sending the form data to `contact.html`. This is probably not what you were intending.